pkgs/servers/fusiondirectory/default.nix

```diff
@@ -0,0 +1,129 @@
+{ lib, stdenv, fetchurl, lndir, schema2ldif, scriptaculous,
+prototypejs, smarty3, smarty3-i18n, gettext, makeWrapper,
+perl, perlPackages, openldap, plugins ? []}: stdenv.mkDerivation rec {
+  name = "fusiondirectory-${version}";
+  version = "1.2.1";
+
+  srcs = [
+    (fetchurl {
+      url = "https://repos.fusiondirectory.org/sources/fusiondirectory/fusiondirectory-${version}.tar.gz";
+      sha256 = "1pzrb3px0qn5rfwh29549lh7fgy46qcx2wlfzhcyax7anhx19ndx";
+    })
+    (fetchurl {
+      url = "https://repos.fusiondirectory.org/sources/fusiondirectory/fusiondirectory-plugins-${version}.tar.gz";
+      sha256 = "02n2wsp0dljw54wdd38yn0214v7kf218iaz0z9qw8ia1x0ffilnb";
+    })
+  ];
+
+  nativeBuildInputs = [ makeWrapper ];
+  phases = [ "unpackPhase" "patchPhase" "installPhase" ];
```

Comment:
It is generally more robust to disable problematic phases rather than explicitly listing phases.  Might not matter for this package specifically, but missing important phases is a source of bugs.